assets/js/components/modal.js

```diff
@@ -235,28 +241,26 @@ const modal = {
     },
 
     // Close the popup
-    close: function (modal) {
+    close: function ( modal ) {
+
+        const $modal = modal.classList.contains('c-modal') ? $(modal) : $(modal).closest(".c-modal");
```

Comment:
Not sure what our current "best practices" policy is regarding the js-prefixed classes for hooking in with JS, but this one doesn't necessarily conform with the idea of only using js-prefixed classes to target HTML elements (as here we look for c-modal rather than js-modal). 

One can debate whether the cost of extra classes (adding js-modal class to the template) is worth the increased transparency when viewing the markup, and I guess what really matters most is a certain consistency in the Official Swiss-Approved Code :)

Reply:
Yeah, using only js-prefixed classes would be nice here but that'll be another day & another PR :)